docs/source/02-Analytics-Query/User-Count/05-At-Least-Two-SP.sql

```diff
@@ -0,0 +1,13 @@
+/*
+**Definition**: number of Users who created their Login.gov account and completed registering MFA as a result of initiating their Login.gov session through that SP.
+
+**SQL Description**:
+
+unlike ``users`` table, any users in ``identities`` have been finished MFA setup.
+*/
+
+\set end_time '''2019-01-01'''
+SELECT
+    (COUNT(identities.user_id) - COUNT(DISTINCT(identities.user_id))) AS at_least_2sp_user_counts
```

Comment:
this will not get the count of users with two or more sps.  for example if there is only one user and he has 3 sps this would return 3 - 1 = 2 users.  Also the description does not match the name of this file (at least 2 sps)

Reply:
@stevegsa fixed